packages/drivers/driver-base/src/documentDeltaConnection.ts

```diff
@@ -406,10 +406,31 @@ export class DocumentDeltaConnection
                 let isWebSocketTransportError = false;
                 try {
                     const description = error?.description;
-                    if (description && typeof description === "object") {
+                    const context = error?.context;
+
+                    if (context && typeof context === "object") {
+                        const statusText = context.statusText.code;
+
+                        // Instead of string-matching the ErrorMessage, Filter the Error Based on the ErrorCode: "DEPTH_ZERO_SELF_SIGNED_CERT"
```

Comment:
This file already does this, I was suggesting we add this comment as a `//TODO: instead of string-matching...` on the other file, where we still need to rely on string matching 😄 .